cmd/notation/registry.go

```diff
@@ -41,7 +41,7 @@ func getRegistryClient(ctx context.Context, opts *SecureFlagOpts, serverAddress
 	return reg, nil
 }
 
-func getRepositoryClient(ctx context.Context, opts *SecureFlagOpts, ref registry.Reference) (notationregistry.Repository, error) {
+func getRepositoryClient(ctx context.Context, opts *SecureFlagOpts, ref registry.Reference, ociImageManifest bool) (notationregistry.Repository, error) {
```

Comment:
Can we add a comment to mention that the `ociImageManifest` only influences on signing logic but not the verifying and listing logic.

Reply:
It's an unexported helper function though. Will add a description comment for this argument.